cddl/intrep-ev.cddl

```diff
@@ -0,0 +1,12 @@
+ev = [
+  condition: [ + ECT ]
+  addition: [ + ECT ]
+]
+
+evs = [
+  condition: [ + ECT ]
+  series: [ + {
```

Comment:
This map syntax for the pair needs key definitions, no? `/ selection: / 0 => [ + ECT ]` for example. Same comment for rv below.

Reply:
It's good like that.  We decided to avoid codepoint aliases here and just use string labels because compression is not needed.

Reply:
Keep in mind that a primary goal of defining an internal representation is to facilitate readability of document prose. It isn't to arrive at the most optimal CDDL. 

Recursive structures are harder to describe IMHO.  The purpose for the 'series: ' label is to make it easier to describe the processing differences (which uses OR semantics where the first match terminates processing) from the 'addition:' (which uses AND semantics.)

Combining ev and evs with nesting seems like its going in the wrong direction.

Actual implementations can always apply these sorts of optimizations if they want to.